fishtest/utils/index_change.py

```diff
@@ -0,0 +1,55 @@
+
+# index_change.py - drop/create indexes
+#
+# Run this script manually to adjust indexes, it could take a few seconds/minutes
+# to run.
+#
+# Ad-hoc changes can be made by adding / commenting out lines as appropriate
+
+
+import os
+import sys
+import pprint
+from pymongo import MongoClient, ASCENDING, DESCENDING
+
+
+db_name='fishtest_new'
+
+# MongoDB server is assumed to be on the same machine, if not user should use
+# ssh with port forwarding to access the remote host.
+conn = MongoClient(os.getenv('FISHTEST_HOST') or 'localhost')
+db = conn[db_name]
+runs = db['runs']
+pgns = db['pgns']
+
+
+
+def printout(s):
+  print s
```

Comment:
Please use python3 `print(...)` and the corresponding import for python2.6+ compatibility